.github/workflows/upgrade-python-requirements.yml

```diff
@@ -13,8 +13,8 @@ jobs:
    call-upgrade-python-requirements-workflow:
     with:
        branch: ${{ github.event.inputs.branch }}
-       team_reviewers: "arbi-bom"
-       email_address: arbi-bom@edx.org  
+       user_reviewers: "sarina"
+       email_address: sarina@axim.org  
```

Comment:
I don't think the email address is needed, it's not used here: https://github.com/openedx/docs.openedx.org/blob/main/.github/workflows/upgrade-python-requirements.yml